### ISSUE TITLE
MVP Offering? [Short-term]

### ISSUE THREAD

**Issue by nelsonic** [bmhco/shared#7]
As we have previously discussed, the "MVP" of setting up a Montessori House is an **_informal_ Playgroup**.

# Highlights

+ [x] €20 for 09:00 - 12:00 (3 hours) = **`€6.67/hour`**
+ [x] Healthy vegan snacks included/provided including those baked/made by the children [under adult supervision].
+ [x] Initially we will open our doors for 2 days per week Mondays and Fridays. (because these appear to be the days where there aren't any other playgroups in our area.
+ [x] Parents will be expected to stay in the space with their child but do not have to directly supervise their child as there will be a Montessori-trained Assistant to help children perform activities. 
+ [x] All parents have to complete an induction [TBD] and sign-off on the rules prior to their child attending.
+ [x] Indoor planned activities will be listed on our website; e.g. Arts and Crafts, Music, Dance, Story Time, etc.
+ [x] Outdoor excursions e.g. Park visits, Library, Museum etc. will be on the `public` calendar. 
+ [x] In the future, parents could leave their child with us and go do errands/work [but not immediately as we need licenses/paperwork for this type of operation]

# Requirements

+ [ ] Tidy the entrance of @home and child-proof anything that our risk assessment concludes _could_ be hazardous. 
+ [ ] Downstairs diaper changing station.
+ [ ] Make our offering `public` on the website.

**Comment by Stephanymtr** [comment]
Hi Nelson, I have a couple of follow-up questions:

### Highlights
- [ ]  €20 for 09:00 - 12:00 (3 hours) = €6.67/hour
- [ ]  Healthy vegan snacks included/provided including those baked/made by the children [under adult supervision].

> How many children are we going to be welcoming for this initial trial ? Considering we will offer snacks, we should have some sort of seating area for all the little ones to sit down to eat together. We have talked about turning the dining room into some sort of Montessori Family-Style Dining space ( and Practical life activities too). Should we create a layout for that room as well? 

- [ ]  Initially we will open our doors for 2 days per week Mondays and Fridays. (because these appear to be the days where there aren't any other playgroups in our area.

> Can/Should we have a training practice morning to simulate welcoming parents into the space  in order to prepare all guides/employees for the real thing ? 

- [ ]  Parents will be expected to stay in the space with their child but do not have to directly supervise their child as there will be a Montessori-trained Assistant to help children perform activities.

> Where will the parents stay? We might need seating area in the library or under the stairs... ?

- [ ]  All parents have to complete an induction [TBD] and sign-off on the rules prior to their child attending.
> I'd suggest creating something similar for all employees as well.
- [ ]  Indoor planned activities will be listed on our website; e.g. Arts and Crafts, Music, Dance, Story Time, etc.
- [ ]  Outdoor excursions e.g. Park visits, Library, Museum etc. will be on the public calendar.
> Will parents be expected to tag along in these excursions ?
- [ ]  In the future, parents could leave their child with us and go do errands/work [but not immediately as we need licenses/paperwork for this type of operation]

### Requirements
- [ ]  Tidy the entrance of @home and child-proof anything that our risk assessment concludes could be hazardous.
> How would you like to child-proof the main floor ? Do a walk though the main floor? Take photos of the areas? 
- [ ]  Downstairs diaper changing station.
> I believe the bathroom from the main floor would be the best area for this.  Did you already have any ideas or should we create a layout as well? I'm thinking we might need some sort of toilet seat with a step stool ladder for the kiddos who are (being) potty trained.
- [ ]  Make our offering public on the website.
> Yay! Should I create a new tab on the website for this ? _I'm about to do some editing on the website and look for themes to elevate the website appearance. Once that's done, I'm happy to create a tab for this if you agree._

**Comment by nelsonic** [comment]
Thanks for the follow-up questions @Stephanymtr 🎉 
I will attempt to answer them in order. 💬 

## How Many Children?

One at a time. So for the purposes of this discussion; **`1`**.
But if there is demand for more children from parents who want to be here in-person (during the drial),
then we can accommodate up to **`10`** (easily), provided the downstairs changing area is created.

We don't want people wondering upstairs (at least not until we have our e-Locks on the bedroom doors.
I'd say **top 2 floors** should be **off-limits** for the time-being. 
Which works well for keeping the **1<sup>st</sup> Floor Tidy** (clutter-free) and spotlessly clean.

The number of children very much depends on: 
1. Their **level of _independence_** of the child(ren) - if children _don't_ need constant attention/supervision when performing a non-dangerous independent task like counting, puzzles, reading, etc. it's easier.
   > **Note**: independence also includes **`toilet`** independence 🚽  
  So we need to capture that in our "register interest form": https://github.com/bmhco/shared/issues/9
2. **How many _adults_** are in attendance - if parents are here, and we have well-defined activities it's much easier.
3. **Availability of additional guides/assistants** - we should have a _minimum_ of 2 adults at all times [other than bathroom breaks] in the work/play room. If we can recruit more guides and determine that there's "enough space" we could accommodate more children. 

## Snacks ?

We will bulk-buy our long-shelf-life ingredients (baking flour, oats, etc.) 
and have supplies on-hand so we can _make_ snacks for _many_ children at a moments' notice. 
It will be good to ***`include`*** the children in the process the way we already do with our own. 
That way they _participate_ in making their own food/snacks.

> **Note**: we need study the kitchen _carefully_ in our risk assessment. #8 
It's already, _relatively_ safe ... but we need a **_written_ system** for handling sharp, breakable & hot objects.

Water and Snacks (e.g: fruit and sugar-free vegan baked items) 
will be available at all times at children's level 
so they don't have to _ask_ when they are hungry. `#independence` 
We just need a decent absorbent rug in front of the child-friendly water station. 💭 

## Family Dining Space

I am **_particularly_ keen** on this one because it helps us **create _community_** not a "transactional" operation.
Please see: https://github.com/bmhco/shared/issues/11

## Opening Times: 2 Days Per Week

The 2-days per week opening is arbitrary but we think it’s a good starting point.
If it goes well we can expand.

## Internal Training

Yes, absolutely, we need to be **100% clear** on our protocols/processes for welcoming other children + parents into our home. We need to set a “start date” so that we have a “deadline” we can work back from: https://github.com/bmhco/shared/issues/13

The handbook will contain sections for the staff but it’s also relevant to parents to be aware of so I think we could safely include _everything_ in the same handbook. All-ears if anyone thinks otherwise.


## Parents Training

We need to create a Parents & Caregivers Handbook: https://github.com/bmhco/shared/issues/12
Parents will need to _read_ the handbook - which will be written in **`EN`** but translated to **`PT`** (by a native **`PT`** speaker).

## Parents That Stay …

Parents will be encouraged to _observe_ their children through the window/door but avoid interfering/interrupting with the child’s focus/learning time.
 Parents can use the facilities e.g. kitchen to make themselves a cup of tea or coffee and are welcome to sit on the L-shaped sofa (under the stairs), library (if it’s not in-use by children/staff) and dining room (also when not in-use). 

## Excusions

We won’t _expect_ parents to go on excursions, but it really depends on _where_ we’re going. e.g. imagine that we take a walk down to the basketball courts, the parents might _want_ to join so they can play with their child. But if they aren unavailable, provided we have the appropriate ratios we can still go. But yes, full-time parents [though vanishingly few in PT] are welcome to join provided they have read the handbook and understand their role is “support” not “authority”.

## Child-proofing …

All the furniture that we have built so far is with children [safety] in mind.
Anything that we deem to be unsafe will be removed as part of the risk assessment https://github.com/bmhco/shared/issues/8

## Toilet Seat for Potty Training / Toilet Independence

1000% see: https://github.com/dwyl/learn-cad-cam-cnc/issues/81
This is _high_ on my list of next projects to complete in the workshop. 


## Public 

Making our Playgroup Proposal ***`public`*** requires the form: https://github.com/bmhco/shared/issues/9 this to avoid chaos.

**Comment by nelsonic** [comment]
This afternoon @iteles' aunt who is a kindergarten teacher visited @home for tea. She remarked that the work/play room is _smaller_ than her classroom which has ***25 children***. She teaches at a public sector primary school and has mixed ages in her class including children who are still in diapers and cannot feed themselves. 🚼

100% not sayin' we will have that many children here, just a reference point. 💭

Montessori work rooms tend to be larger and sparsely populated mostly because they are ***private*** (paid) so can _afford_ more space. 💰

**Comment by iteles** [comment]
@nelsonic Sorry, I didn't hear the comment but from the remainder of your comment it made it sound like she said her classroom was smaller and not the other way around?

A couple of photos of her classroom could be useful!

Also, traditional classrooms have very few bateristas actually ser out for kids constantly.

**Comment by Stephanymtr** [comment]
Thank you for answering all my questions! I have a better understanding of where we stand with all the ideas. I will comment/update on the new issues you have created for each subject. We should definitely have a list of things we need before we open, so that we don't get distracted with all the other awesome ideas we can implement in the future.

**Comment by nelsonic** [comment]
Yeah, we definitely need to focus. 🎯
Look forward to reading your feedback + follow-up questions. 👌

**Comment by nelsonic** [comment]
a month later ... ⏳ 
@iteles could you please review the 3 top comments in this issue 
(not just what your Aunt said about the room ...)
and let us know your thoughts. 🙏 
Thanks.

**Comment by iteles** [comment]
@nelsonic @iteles Of course. This has not been my focus due to family matters and urgent accounting admin these past 2 months but absolutely is my priority for March. I'm on it!

**Comment by iteles** [comment]
@nelsonic @Stephanymtr Re-focussing.

What age group are we thinking of for this? 6 months + makes perfect sense because we have a 👶 in this age group of course.

For 👦 , the vast majority of the 3-6 age group is in preschool/nursery in Portugal, so if we want some older kids to come along too (so everyone is not just younger than A), would it make sense to also do some afternoon playgroup time once traditional preschool/nap time is finished (15h30 in most cases)?

**Comment by nelsonic** [comment]
I'd say `18 months - 6 years` (long-term). 👶 👧 
But for now Max Age should be `4 years old` as a `5-6 year-old` with our two would be unwieldy. 💭 
I don't think we would have _any_ shortage of candidate children for the 2 - 4 age group.
There is such a shortage of good quality pre-school as you know. 
The best way to research this is to _ask_ people to fill-in a small survey. #9 
We could put something together in less than an hour. ⏳ 
We just need to agree what data we want to collect (keeping it to minimum) and then put up the form.
e.g: 

<img width="598" alt="image" src="https://github.com/bmhco/shared/assets/194400/15d6e744-c9b4-40cc-ae71-4cc9cee74fad">

`HTML`:

```html
<fieldset class="ml-4 mt-4">
  <legend class="sr-only">Plan</legend>
  <div class="space-y-5">
    <div class="relative flex items-start">
      <div class="flex h-6 items-center">
        <input id="small" aria-describedby="small-description" name="plan" type="radio" checked class="h-4 w-4 border-gray-300 text-green-600 focus:ring-green-600">
      </div>
      <div class="ml-3 text-sm leading-6">
        <label for="small" class="font-medium text-gray-900">09:00 - 12:00 Manhã | Mornings</label>
        <p id="small-description" class="text-gray-500">Inclui lanche saudável | Includes healthy snacks</p>
      </div>
    </div>
    <div class="relative flex items-start">
      <div class="flex h-6 items-center">
        <input id="medium" aria-describedby="medium-description" name="plan" type="radio" class="h-4 w-4 border-gray-300 text-green-600 focus:ring-green-600">
      </div>
      <div class="ml-3 text-sm leading-6">
        <label for="medium" class="font-medium text-gray-900">13:00 - 16:00 Tarde | Afternoon</label>
        <p id="medium-description" class="text-gray-500">Inclui lanche saudável | Includes healthy snacks</p>
      </div>
    </div>
    <div class="relative flex items-start">
      <div class="flex h-6 items-center">
        <input id="large" aria-describedby="large-description" name="plan" type="radio" class="h-4 w-4 border-gray-300 text-green-600 focus:ring-green-600">
      </div>
      <div class="ml-3 text-sm leading-6">
        <label for="large" class="font-medium text-gray-900">09:00 - 16:00 Dia Completo | Full Day</label>
        <p id="large-description" class="text-gray-500">Inclui almoço e lanche saudável | Includes lunch and healthy snacks</p>
      </div>
    </div>
  </div>
</fieldset>
```

Seriously we could put together the registration form _very_ fast. (wish I'd just done it in _February_... ⏳ 🙃)

**Comment by iteles** [comment]
My brain is back online 🎉  2 years post partum seems to be the magic time for me.

I have been thinking about this a lot because for it to work in my mind I needed to have some idea of activities and equipment too.  _I also needed to come to terms with being 'authentically Montessori' or 'Montessori inspired' (which earns some scoffs). This is a separate issue.

***The key for me with the MVP is to open _with_ a coworking space offering in the current 'dining room' for parents to be able to have some work/downtime***. 
They need to be physically there because (as @nelsonic says in the OP, the alternative requires a lot of extra paperwork) _but_ the differentiator here is without a doubt the co-working area.

Some bullet points:
+ 1-2 mornings a week (for now it will have to be Tuesdays and Thursdays because this is when we have help)
+ Will open a separate issue on pricing but keen to offer bulk pricing for people who want to come more often
+ 3months - 3 years for now (for _weekday mornings_)
+ Starting by the end of January 2024


I am tempted to shortly after this add a weekday afternoon and even a Saturday morning (but this may be more of a 'true playgroup' without the coworking) for a broader age range.


## Separate issues to be open
+ [ ] Materials/themes
+ [ ] Pricing
+ [ ] How to publicise - the inability to use instagram for this has ben a big factor in my mental blockers but this is definitely a separate issue
+ [ ] What needs to be done in the immediate term

**Comment by nelsonic** [comment]
Inês, you've been doing _many_ other things in the time that has passed including **_two_ Montessori courses**. 2‍⃣ 
It's not that your "brain" wasn't working ... you _chose_ not to focus on this. 😕 
You've done _many_ Landim visits and meetings where your brain worked just fine. 🧠 ✅ 

> **Note**: definitely not saying "post partum" isn't a "thing". 
> Just not a good _excuse_ for why there has been _zero_ progress on _this_ issue in 16 months.
> <img width="368" alt="Image" src="https://github.com/user-attachments/assets/13ce45ae-b234-4017-af5f-1d02fcd0a41f" />

I described the "Playgroup" can coexist seamlessly with our original plan including a sample timetable in: 
[https://github.com/dwyl/home/?#montessori-playgroup-centro-de-actividades](https://github.com/dwyl/home/blob/fcedbe7415cd843535b464ec9b812c064710b225/README.md#montessori-playgroup-centro-de-actividades) 🔗 
Which you reviewed and merged 8 months ago. ⏳ https://github.com/dwyl/home/pull/277
You've had all that time to ask follow-up questions. 
Only now that `Mika` is _leaving_ are you waking up to the urgency on this? 😖 

We already have an Attic _Full_ "activities and equipment" https://github.com/dwyl/home/issues/221#issuecomment-1465168243 so you already have "some idea" ... 💡 
99.9% of parents have _no idea_ who [Maria Montessori](https://en.wikipedia.org/wiki/Maria_Montessori) is/was or any clue what "authentically Montessori" means. 🤷‍♀ 
Maria Montessori's original [Casa dei Bambini](https://en.wikipedia.org/wiki/Maria_Montessori#Casa_dei_Bambini) was ***FREE*** for underprivileged children of factory workers.
Unless you make it ***FREE***, it's not "authentically Montessori". 🙃 

Please don't get hung-up on anyone `else`'s "scoffs". They are [armchair quarterbacks](https://en.wiktionary.org/wiki/armchair_quarterback). `#ignore`

# Bullet Points

+ **Tuesdays & Thursdays** seem good. Is `S` on-board with helping?

+ **Pricing** should be **`€4/h`** to start just to help cover costs but not be _too expensive_.
  > **`€4/h`** with 3 kids covers the **marginal costs** and gets the ball rolling. 
  > This _will_ be revised higher e.g. the `€6.67` described in the OP ⤴ and we need to make that clear to people. 
  > We have more than enough cash (from my contracting) to cover **fixed overheads** in the short-term 
  > As I described in the OP 2+ _years_ ago, the money isn't - never was - the issue! 

+ **Bulk pricing** makes sense once you've run the Playgroup for a couple of months.
  It should be done via `App` (which can be built in a couple of days) so you/we aren't managing spreadsheets!

+ **Age Range** is very much dependent on the "help" you can recruit. ... 
unless _you_ want to change a lot of diapers! 💩 
With `Mika` leaving at the end of `January` (😢) we need to find another good person to help. 
Ideally someone who either _already_ has `Montessori` knowledge/training or who wants to do a course. 
Happy to pay for their course same way we (I) did for `Tété` in 2022 ... (⏳🤯) 
I will _very_ happily be a "guide" to _our_ children and a handful of others for a few hours a day/week.
But I _must_ focus on the 

+ **How to publicise** - this won't be difficult at all. You have been _completely_ brainwashed by Mark Zuckerberg into thinking that `Instagram` is a _necessity_ for promoting anything. It _never_ was and isn't today.
You only need to think about what the _capacity_ is and then realise that we can _easily_ get **20+ children** by putting the word out. You/we _definitely_ don't need _thousands_ of "followers" to get the message out. 

***Start*** by updating the **landing page** of the website to describe the offering 
with a few photos of materials and locations:
e.g: play room and nearby park(s) we visit with children (with full adult supervision).

Then print off an **`A5` sheet** of **paper** with a **`QR Code`** and website address and put it on the **`Mimo Natura` _physical_ notice board**. That will generate 100+ "leads" in a month. Collect data from interested parents on the landing page. We've already had _several_ inquiries from the Website without any decent effort to make a landing page. 📲 

+ **_Immediate_ Term**: **tidy the house**. Remove _all_ non-essential items from _all_ surfaces and floors.
   This hasn't changed in the **2+ years** since this issue was opened. 

  > I feel like a firefighter chasing an arsonist when it comes to tidying the house. 😫 
  I spend _weeks_ of my life building storage e.g: https://github.com/dwyl/home/issues/311 ⏳ 
  when that time could _easily_ be invested elsewhere if you just didn't _buy_ as much stuff!! 😖 
  We have _shops_ and online that delivers next-day! Why do we need to have so many _spare_ cleaning products? 🤷‍♂ 
  Why do we _need_ to have an **attic _full_** of unused toys/learning-materials? 🤷‍♂ 
  Why do we _need_ to have an _ocean_ of "spare" clothes for the children taking up a whole room?! 👚 
  Your _refusal_ to build a _single_ **documented `system`** for the house continues to boggle my mind. 🤯 

Apologies to anyone reading this thinking I'm coming across as "negative". ❎ 
I'm just sick of _waiting_ ... ⏳ 😢 
Nothing kills dreams more tragically than **procrastination**. 💀
